### PR TITLE
The first feature review

### PR DESCRIPTION
**Pull request by Wh1teCl0ud**
In this feature branch I've added:
- .gitignore file and filled it properly;
- An HTML&CSS linter;
- Description in README file;
- Index.html; 
- styles.css; 
- test.md.

### CODE REVIEW

File: .gitignore/test.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```
Comment (jasonfelice) [inline .gitignore/test.md:1]: - [ ] Kindly note that `.gitignore` is a file, not a folder. To fix this:
  - Delete this folder
  - Add a `.gitignore` file in this repository
  - Add `test.md` inside the `.gitignore` file.

image

**Review by jasonfelice — CHANGES_REQUESTED** [review]
Hi @Wh1teCl0ud, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Good commit messages✔️
No linter errors ✔️ 

## Required Changes ♻️
- [ ] Please update the PR description to summarize what you've done in this branch. 😄 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Wh1teCl0ud** [comment]
I updated the initial comment. If every time I need to make a new comment, tell me.

**Review by awais-amjed — APPROVED** [review]
# STATUS: APPROVED 🥳

Hi @Wh1teCl0ud , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations on implementing the required changes! 🎉👏🥇✨

## Highlights

- [x] Descriptive Pull Request ✔️
- [x] No Linter issues ✔️
- [x] Good code organization ✔️ 

gif

## Optional suggestions

- [ ] Kindly note that when you add a file to `.gitignore` that has already been pushed to GitHub, it does not delete it from GitHub automatically, so you will need to delete `test.md` manually. But once you add a file to `.gitignore` it won't be pushed to GitHub after that 🤗
- [ ] Also kindly use the [Readme Template from Microverse](https://github.com/microverseinc/readme-template) for your Readme file and customize it to your project 🤗

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**